core/src/json_schema.rs

```diff
@@ -282,6 +363,297 @@ pub mod simplified {
 
 #[cfg(test)]
 mod unit {
+    macro_rules! remove_multiple_test {
+        ($func_name:ident, expected = $expected:expr, input = $input:expr) => {
+            #[test]
+            fn $func_name() {
+                let res = $input.remove_multiple().unwrap();
+
+                assert_eq!(res, $expected);
+            }
+        };
+    }
+
+    // Testing TryFrom<json_schema::Schema> for json_schema::simplified::Atomic
```

Comment:
This comment should either be removed or moved to be directly above `mod try_from_test`.

Reply:
whoops!